build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     ext.wordPressUtilsVersion = '2.2.0'
     ext.wordPressLoginVersion = '0.0.8'
     ext.detektVersion = '1.15.0'
-    ext.gutenbergMobileVersion = 'v1.66.0'
+    ext.gutenbergMobileVersion = 'v1.68.0-alpha1'
```

Comment:
We might get a conflict after GB-mobile version `1.67.0` is merged into `develop` and we try merging the branch `gutenberg/after_1.67.0`. In that case, we might need to create a new tag `v1.68.0-alpha2` to point to a later GB-mobile commit so it includes changes from `1.67.0` and the changes from https://github.com/wordpress-mobile/gutenberg-mobile/pull/4290.